code-challenges.test.js

```diff
@@ -15,16 +15,56 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("fibonacci", () => {
+    it(" takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.", () =>{
+        expect(fibonacci(6)).toEqual([1, 1, 2, 3, 5, 8])
+        expect(fibonacci(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+})
+
+// ● fibonacci ›  takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+//ReferenceError: fibonacci is not defined
+
+
+
+
 // Example input: 6
 // Expected output: [1, 1, 2, 3, 5, 8]
-
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
 // b) Create the function that makes the test pass.
 
-
+const fibonacci = (num) => {
```

Comment:
This approach definitely works.  This could also be refactored so that you don't need the filter on line 50 by setting arr1 to [1,1] and your iteration for i < num.